src/test/graph_export_test.cpp

```diff
@@ -246,6 +246,52 @@ const std::string expectedEmbeddingsGraphContents = R"(
     }
 )";
 
+const std::string expectedImageGenerationGraphContents = R"(
+input_stream: "HTTP_REQUEST_PAYLOAD:input"
+output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+
+node: {
+  name: "ImageGenExecutor"
+  calculator: "ImageGenCalculator"
+  input_stream: "HTTP_REQUEST_PAYLOAD:input"
+  input_side_packet: "IMAGE_GEN_NODE_RESOURCES:pipes"
+  output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+  node_options: {
+      [type.googleapis.com / mediapipe.ImageGenCalculatorOptions]: {
+          models_path: "./"
+          target_device: "GPU"
+          plugin_config: '{"NUM_STREAMS":14,"CACHE_DIR":"/cache"}'
+          max_resolution: "3000x4000"
+          default_resolution: "300x400"
+          max_number_images_per_prompt: 7
+          default_num_inference_steps: 2
+          max_num_inference_steps: 3
+      }
+  }
+}
+
+)";
+
+const std::string expectedImageGenerationGraphContentsDefault = R"(
+input_stream: "HTTP_REQUEST_PAYLOAD:input"
+output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+
+node: {
+  name: "ImageGenExecutor"
+  calculator: "ImageGenCalculator"
+  input_stream: "HTTP_REQUEST_PAYLOAD:input"
+  input_side_packet: "IMAGE_GEN_NODE_RESOURCES:pipes"
+  output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+  node_options: {
+      [type.googleapis.com / mediapipe.ImageGenCalculatorOptions]: {
+          models_path: "./"
+          target_device: "CPU"
```

Comment:
Wouldn't it be better if we set this default in proto?

Reply:
I think this is specific parameter, that makes it easy for an admin to see where the models will be loaded. Default in proto is also good, as a fallback. I liked this approach and copied it from rerank and embeddings behavior. What do you think about it?